NEWS.md

```diff
@@ -1,80 +1,112 @@
 # ggplot2 3.1.0.9000
 
-* `geom_rug()` gains a `length` option to allow for changing the length of the rug lines. (@daniel-wells, #3109)
+This is a minor release with a main emphasis on internal changes to make ggplot2 
+faster and more consistent. The few semi-breaking changes there are will only 
+affect the aesthetics of the plot (but not break code) or, potentially, 
+extension developers if they have relied on internals that have been changed. 
+This release also sees the addition of Hiroaki Yutani (@yutannihilation) to the 
```

Comment:
Thanks for this sentence, but I think we have not described the change of the core developer team in NEWS so far. Am I really a news to users...?

Reply:
We may not have been doing it before, but I think it belongs in a NEWS file...

Reply:
I see.

Reply:
Agreed!